contrib/builder/deb/aarch64/generate.sh

```diff
@@ -97,18 +99,31 @@ for version in "${versions[@]}"; do
 			;;
 	esac
 
-	echo "# Install Go" >> "$version/Dockerfile"
-	echo "# aarch64 doesn't have official go binaries, so use the version of go installed from" >> "$version/Dockerfile"
-	echo "# the image to build go from source." >> "$version/Dockerfile"
+	case "$suite" in
+		jessie)
+			echo "# Install Go (hack: for now use pre-compiled tarball from Hypriot)" >> "$version/Dockerfile"
```

Comment:
This should work. It will download and build 1.4, and then use it to bootstrap GO_VERSION, then it will remove 1.4. 

Note: you need to add gcc to the packages, and you will still need to set the path / gopath after this, but the go bin should be in /usr/src/go/bin/go

```sh
export GO_VERSION=1.7.4

mkdir /usr/src/go-1.4 && curl -fsSL https://storage.googleapis.com/golang/go1.4-bootstrap-20161024.tar.gz | tar -v -C /usr/src/go-1.4  -xz --strip-components=1 \
        && cd /usr/src/go-1.4/src && ./make.bash

mkdir /usr/src/go && curl -fsSL https://golang.org/dl/go${GO_VERSION}.src.tar.gz | tar -v -C /usr/src/go -xz --strip-components=1 \
        && cd /usr/src/go/src \
        && GOOS=linux GOARCH=arm64 GOROOT_BOOTSTRAP="/usr/src/go-1.4" ./make.bash \
        && rm -rf /usr/src/go-1.4
```